addons/website_sale/models/product_product.py

```diff
@@ -309,116 +273,206 @@ def _prepare_gmc_price_info(self):
         if combination_info['prevent_zero_price_sale']:
             return {}
 
-        gmc_info = {
-            'price': utils.gmc_format_price(
+        price_info = {
+            'price': utils.product_feed_format_price(
                 combination_info['list_price'], combination_info['currency'],
             ),
         }
         # sales/promo/discount/etc.
         if combination_info['has_discounted_price']:
-            gmc_info['sale_price'] = utils.gmc_format_price(
+            price_info['sale_price'] = utils.product_feed_format_price(
                 combination_info['price'], combination_info['currency'],
             )
             start_date = combination_info['discount_start_date']
             end_date = combination_info['discount_end_date']
             if start_date and end_date:
-                gmc_info['sale_price_effective_date'] = '/'.join(
-                    map(utils.gmc_format_date, (start_date, end_date)),
+                price_info['sale_price_effective_date'] = utils.format_sale_price_effective_date(
+                    start_date, end_date, format=format,
                 )
 
-        # Note: Google only supports a restricted set of unit and computes the comparison prices
-        # differently than Odoo.
+        # Note: Google and Meta only supports a restricted set of unit and computes
+        # the comparison prices differently than Odoo.
         # Ex: product="Pack of wine (6 bottles)", price=$65.00, uom_name="Pack".
         #   - in odoo: base_unit_count=6.0, base_unit_name="750ml"
         #       => displayed: "$10.83 / 750ml"
-        #   - in google: unit_pricing_measure="4500ml", unit_pricing_base_measure="750ml"
+        #   - in google/meta: unit_pricing_measure="4500ml", unit_pricing_base_measure="750ml"
         #       => displayed: "$10.83 / 750ml"
-        if (
-            combination_info.get('base_unit_name')
-            and self.base_unit_count
-            and (match := const.GMC_BASE_MEASURE.match(
-                combination_info['base_unit_name'].strip().lower()
-            ))
-        ):
-            base_count, base_unit = match['base_count'] or '1', match['base_unit']
-            count = self.base_unit_count * int(base_count)
-            if (
-                base_unit in const.GMC_SUPPORTED_UOM
-                and not float_is_zero(count, precision_digits=2)
-            ):
-                gmc_info['unit_pricing_measure'] = (
-                    f'{float_round(count, precision_digits=2)}{base_unit}'
+
+        base_unit_name = combination_info.get('base_unit_name')
```

Comment:
nit: remove extra empty line
```suggestion
        #       => displayed: "$10.83 / 750ml"
        base_unit_name = combination_info.get('base_unit_name')
```